_includes/data_and_docs.html

```diff
@@ -0,0 +1 @@
+<img src="{{site.baseurl}}/public/img/icons/file-text-o.svg" class="u-padding-right icon-small"/>Data and documentation
```

Comment:
Two questions here:

* I thought one of the purposes of the refactor was the make `svg`s styleable, what do you think about having the actual svg included in the partial? Then the fill color can be controlled by a variable.

* In a few places I noticed we are directly using this image tag because the text accompanying the svg is different. What do you think about passing an optional variable to the include that could change the text if necessary, keeping 'Data and documentation' as the default? It would look something like this:

```
{ assign content = include.content || "Data and documentation" }
<img src="{{site.baseurl}}/public/img/icons/file-text-o.svg" class="u-padding-right icon-small"/>{% content %}
```

and it would get called like this:

`{% include data_and_docs.html content="Some other text" %}`

Reply:
Great suggestions, @el-mapache. I [committed some changes](https://github.com/18F/doi-extractives-data/pull/2841/commits/674709837d6b2d14bf19e4d74161ae8b904e9322) to implement them. Please let me know your thoughts.

Reply:
looks great @brentryanjohnson !